### PR TITLE
chore: bump sdk-bridge package version to support infinite token approval

### PR DESCRIPTION
**Pull request by Imti**
Closes https://github.com/nomad-xyz/nomad-app/issues/110

### CODE REVIEW

**Comment by ErinHales** [comment]
Hm, shoot.  Getting this error.  The max amount for approval is overflowing, I'm not sure why.  Maybe we need to decrease the value a bit

Screen Shot 2022-06-02 at 8 43 35 PM

**Review by ErinHales — CHANGES_REQUESTED** [review]
Above comment needs to be resolved

File: src/views/Transfer/Transfer.pending.vue [hunk 1]
```diff
@@ -5,9 +5,6 @@
       <n-text class="text-[20px] uppercase text-lg font-medium text-white my-5">
         Sending transaction
       </n-text>
-      <n-text class="opacity-70 mb-3">
-        Please continue to your wallet to<br />approve your bridge transaction
-      </n-text>
```
Comment (ErinHales) [inline src/views/Transfer/Transfer.pending.vue:10]: We need to keep this text actually. Normally there are 2 steps to a bridge transfer: `approve` and `send`.  If the user has already approved enough, then they still need to head to their wallet to approve the send transaction. The wording is a bit confusing haha, hopefully that makes sense

**Review by ErinHales — CHANGES_REQUESTED** [review]

**Comment by Imti** [comment]
Closing in favor of https://github.com/nomad-xyz/nomad-app/pull/373

**Pull request closed without merging**